linter/whitelists/de.txt

```diff
@@ -382,3 +382,10 @@ zeitintensiver
 Zielseiten
 Zugangsdaten
 Übersetzungsfelder
+Mietangebote
+OpenImmo
```

Comment:
`OpenImmo` und `Wodis` scheinen mir nicht spezifisch für Deutsch zu sein, können wir die in die `default.txt` verschieben?

Reply:
Done